cdap-docs/examples-manual/source/examples/sport-results.rst

```diff
@@ -0,0 +1,212 @@
+   - It constructs a partition filter for the input using the league as the only condition, and instantiates
+     the ``results`` dataset with arguments that contain this filter.
+   - It constructs an output partition key for the new partition, and instantiates the ``totals`` dataset
+     with arguments that specify this partition key.
+
+.. literalinclude:: /../../../cdap-examples/SportResults/src/main/java/co/cask/cdap/examples/sportresults/ScoreCounter.java
+    :language: java
+    :lines: 60-86
+
+After the MapReduce has completed, the ``onFinish()`` method finalizes the addition of the new partition.
+Note that this is only needed due to a current limitation in the ``PartitionedFileSet`` implementation: itself
+should add the partition in the output committer of its output format (this will be addressed in a future CDAP release):
+
+.. literalinclude:: /../../../cdap-examples/SportResults/src/main/java/co/cask/cdap/examples/sportresults/ScoreCounter.java
+    :language: java
+    :lines: 88-94
+
+It is worth mentioning that nothing else in ``ScoreCounter`` is specifically programmed to use file partitions.
+Instead of ``results`` and ``totals``, it could use any other dataset as long as the key and value types match.
+
+Building and Starting
+=====================
+
+- You can either build the example (as described `below
+  <#building-an-example-application>`__) or use the pre-built JAR file included in the CDAP SDK.
+- Start CDAP, deploy and start the application and its component as described below in 
+  `Running CDAP Applications`_\ .
+  Make sure you start the Service as described below.
+- Once the application has been deployed and started, you can `run the example. <#running-the-example>`__
+
+Running CDAP Applications
+=========================
+
+.. |example| replace:: FileSetExample
+
+.. include:: /../../developers-manual/source/getting-started/building-apps.rst
+   :start-line: 11
+
+Running the Example
+===================
+
+Starting the Service
+--------------------
+
+Once the application is deployed:
+
+- Click on ``SportResults`` in the Overview page of the CDAP Console to get to the
+  Application detail page, click ``UploadService`` in the *Service* pane to get to the
+  Service detail page, then click the *Start* button; or
+- From the Standalone CDAP SDK directory, use the Command Line Interface:
+
+  .. list-table::
+    :widths: 20 80
+    :stub-columns: 1
+
+    * - On Linux:
+      - ``$ ./bin/cdap-cli.sh start service SportResults.UploadService``
+    * - On Windows:
+      - ``> bin\cdap-cli.bat start service SportResults.UploadService``
+
+Uploading Game Results
+----------------------
+
+First we will upload some CSV files into the ``results`` dataset. For example, to upload the results
+for the 2012 season of the NFL::
+
+  cdap-cli.sh call service SportResults.UploadService PUT leagues/nfl/seasons/2012 body:file resources/nfl-2012.csv
```

Comment:
Is this path going to work? I'm thinking it should be based on the SDK home (though awfully long):
```
 ``$ ./bin/cdap-cli.sh call service SportResults.UploadService PUT leagues/nfl/seasons/2012 body:file examples/SportResults/resources/nfl-2012.csv``
```